api/src/controllers/forms.js

```diff
@@ -51,6 +56,9 @@ const listFormsJSON = forms => {
 
 module.exports = {
   list: (req, res) => {
+    if (!req.userCtx) {
+      return serverUtils.notLoggedIn(req, res);
+    }
```

Comment:
I've made auth required. This feels like a regression but a test asserts that no auth fails: https://github.com/medic/cht-core/blob/master/tests/integration/medic-collect/medic-collect.spec.js#L41